java/src/jmri/implementation/JmriConfigurationManager.java

```diff
@@ -226,7 +226,8 @@ public boolean load(URL url, boolean registerDeferred) throws JmriException {
         }
         // make this url the default "Save Panels..." file
         JFileChooser ufc = jmri.configurexml.StoreXmlUserAction.getUserFileChooser();
-        ufc.setSelectedFile(new File(url.getFile()));
+        ufc.setSelectedFile(new File(url.getPath()));
```

Comment:
Pass the URI, not a String; it’s the string conversion that is causing this issue. Use [FileUtil.utlToURI](http://www.jmri.org/JavaDoc/doc/jmri/util/FileUtil.html#urlToURI-java.net.URL-) if needed. Also close and reopen PRs to force a rebuild instead of adding spaces to files.

Reply:
Done! (Not a clue how to close & reopen PRs; in the past doing that (via the website) has opened a new PR losing all the comments and commit history of the prior PR). The new GitHub Desktop app is almost as confusing as git… I've got to work out a cookbook procedure for using it.)

Reply:
BTW: The url.getFile() method did fix the problem; the saved file name had spaces instead of %20's in it.